### PR TITLE
terminal: allow wallet unlocker services REST calls

### PR DESCRIPTION
**Pull request by guggero**
Fixes https://github.com/lightninglabs/lightning-terminal/issues/805.

When lnd isn't marked as fully ready yet, we didn't allow any REST methods through. But to allow creating a wallet through REST, we also need to allow these methods.

### CODE REVIEW

**Review by ellemouton — APPROVED** [review]
Noice 🙏

**Comment by guggero** [comment]
I keep forgetting that we now also have file based release notes in `litd`. Added a new file.

File: terminal.go [hunk 1]
```diff
@@ -246,11 +267,17 @@ func (g *LightningTerminal) Run() error {
 	// URI as soon as LND can accept the call, i.e. when the lnd sub-server
 	// is in the "Wallet Ready" state.
 	lndOverride := func(uri, manualStatus string) (bool, bool) {
```
Comment (ViktorTigerstrom) [inline terminal.go:269]: Nit: The comment above here for `lndOverride` is maybe a bit misplaced now that `/lnrpc.State/GetState` URI has been moved to a map. I'm fine to keep it as is if you think that the comment is still ok here :).

**Review by ViktorTigerstrom — APPROVED** [review]
Awesome, thanks for the fix :pray:! LGTM :rocket:

**Pull request merged**